backend/src/binaryInstaller.ts

```diff
@@ -40,33 +40,36 @@ class BinaryInstaller {
 
   async installBinary(installer: Installer) {
     return new Promise(async (resolve, reject) => {
-      var tmpDir = tmp.dirSync({ unsafeCleanup: true, keep: true })
-
       // Stop and remove old binaries
-      await this.migrateBinaries(installer.binaryPath())
+      await this.migrateBinaries(installer.binaryPathCLI())
 
       // Download and install binaries
-      await this.download(installer, tmpDir)
+      await this.download(installer)
 
       const commands = new Command({ onError: reject, admin: true })
 
       if (environment.isWindows) {
         if (!existsSync(environment.binPath)) commands.push(`md "${environment.binPath}"`)
-        commands.push(`move /y "${installer.tempFile}" "${installer.binaryPath()}"`)
-        commands.push(`icacls "${installer.binaryPath()}" /T /C /Q /grant "*S-1-5-32-545:RX"`) // Grant all group "Users" read and execute permissions
+        commands.push(`move /y "${installer.tempFile}" "${installer.binaryPathCLI()}"`)
+        commands.push(`move /y "${installer.tempFileMuxer}" "${installer.binaryPathMuxer()}"`)
+        commands.push(`move /y "${installer.tempFileDemuxer}" "${installer.binaryPathDemuxer()}"`)
+        commands.push(`move /y "${installer.tempFileConnectd}" "${installer.binaryPathConnectd()}"`)
```

Comment:
So instead of moving these, let's leave them in their source directory and just `ln -l` them into the installation location.